core-web/libs/block-editor/src/lib/extensions/ai-content-prompt/ai-content-prompt.component.scss

```diff
@@ -0,0 +1,29 @@
+@use "variables" as *;
+
+form {
+    padding-left: 1.875rem;
```

Comment:
We have spacing variables, check here: core-web/libs/dotcms-scss/shared/_spacing.scss